controller/eks-cluster-config-handler.go

```diff
@@ -108,15 +109,22 @@ func (h *Handler) recordError(onChange func(key string, config *eksv1.EKSCluster
 			return config, err
 		}
 		if err != nil {
+
 			if !strings.Contains(err.Error(), "currently has update") {
-				// the update is valid in that the controller should retry but there is
-				// no actionable resolution as far as a user is concerned. An update
-				// that has either been initiated by eks-operator or another source is
-				// already in progress. It is possible an update is not being immediately
-				// reflected in upstream cluster state. The config object will reenter the
-				// controller and then the controller will wait for the update to finish.
+				// The update is valid in that the controller should retry but there is no actionable resolution as far
+				// as a user is concerned. An update that has neither been initiated by the eks-operator or another
+				// source is in already in progress. It is possible an update is not being immediately reflected in
+				// upstream cluster state. The config object will reenter the controller and then the controller will
+				// wait for the update to finish.
 				message = err.Error()
 			}
+
+			messageNoMeta, err := removeErrorMetadata(message)
+			if err != nil {
+				logrus.Errorf("Error removing metadata from failure message: %s", err.Error())
+			} else {
+				message = messageNoMeta
+			}
```

Comment:
nit: simplify the logic 
```suggestion
			} 
			message = messageNoMeta
```

Reply:
I believe my logic is correct here because we don't want the message to be set to `messageNoMeta` if the metadata removal returned an err

Reply:
I see. yeah, it makes sense. :)